GameFighter/Models/Warrior.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace GameFighter
+{
+    public class Warrior
+    {
+        public int Health { get; private set; }
+        public int Attack { get; protected set; }
+        public bool Alive { get => Health > 0 ? true : false; }
```

Comment:
`Health > 0` returns boolean value, so you don't need to use ternary operator here.
We discussed that it's better to use 'Is' or 'Has' prefixes for identifiers (fields, properties) which return boolean value